docs/content/en/docs/how-tos/buildpacks/_index.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "Building Artifacts with CNCF Buildpacks"
+linkTitle: "Buildpacks"
+weight: 100
+---
+
+This page describes building Skaffold artifacts with [buildpacks](https://buildpacks.io/).
+Buildpacks enable building language-based containers from source code, without the need for a Dockerfile.
+
+To use buildpacks with Skaffold, please install the following tools:
+
+* [pack](https://buildpacks.io/docs/install-pack/)
+* [docker](https://docs.docker.com/install/)
+
+## Tutorial
+
+Clone the Skaffold buildpacks [example](https://github.com/GoogleContainerTools/Skaffold/blob/master/examples/buildpacks/) for sample code.
```

Comment:
Lets try to add a one line description on what the example does so that user's know what they are installing.
e.g. https://cloud.google.com/go/getting-started/tutorial-app

```suggestion
In this tutorial, you will deploy a single go based file app on a pod.
Clone the Skaffold buildpacks [example](https://github.com/GoogleContainerTools/Skaffold/blob/master/examples/buildpacks/) for sample code.
```